src/test/java/dev/openfeature/sdk/ImmutableStructureTest.java

```diff
@@ -10,11 +10,7 @@
 import java.util.Map;
 import java.util.Set;
 
-import static org.junit.jupiter.api.Assertions.assertArrayEquals;
```

Comment:
IDE tidied this, but I could undo it.

Reply:
Doesn't bother me.

Reply:
Personally, I am not a fan of wildcard imports. And the IDE should have the option to disallow this, for example, in idea [1].

Anyway, this could be a good code quality check if agree on the standard pracitce 

[1] - https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

Reply:
Feel free to add this to whatever check you please, here and in the contrib. I know some people don't like this, as long as we're consistent I don't mind.